scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/Scheduler.java

```diff
@@ -243,6 +243,13 @@ public boolean test(File directory) {
           .help("start time of last scan")
           .register();
 
+  private static final Gauge loadingRunDirectoryStatus =
+      Gauge.build()
+          .name("miso_runscanner_run_directory_status")
```

Comment:
I don't think a 'status' of 0 or 1 (or even 'true' or 'false') is very clear. Calling this 'run_directory_valid' (and updating the help text as well) might be a little clearer.